pom.xml

```diff
@@ -168,7 +168,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
@HDouss this is not allowed in this project. we must be dependency-free. there is a comment about it in `pom.xml`, I believe